README.md

```diff
@@ -154,6 +154,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [BetterTouchTool](http://blog.boastr.net/)
   - [BibDesk](http://bibdesk.sourceforge.net/)
   - [Billings Pro Server Admin](https://www.marketcircle.com/billingspro/download/billingspro-server/)
+  - [Bitchx](https://github.com/TroyWilson1/)
```

Comment:
The url should be https://github.com/Bitchx - 

Sorry, I'm still learning how this works.